modules/nf-core/bedtools/shift/main.nf

```diff
@@ -0,0 +1,37 @@
+process BEDTOOLS_SHIFT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::bedtools=2.30.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bedtools:2.30.0--hc088bd4_0' :
+        'quay.io/biocontainers/bedtools:2.30.0--hc088bd4_0' }"
+
+    input:
+    tuple val(meta), path(bed)
+    path  sizes
```

Comment:
```suggestion
    tuple val(meta2), path(sizes)
```
I also think `fai` would be a better input name here

Reply:
I kind of copied everything from "bedtools slop" module

Reply:
That module hasn't been updated in a while 😁

Reply:
Actually, why "fai" or "sizes" must have meta2? Trying to understand...